Lanchonete/LanchoneteTests/Managers/IngredientCellManagerTests.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  IngredientCellTests.swift
+//  LanchoneteTests
+//
+//  Created by vera.dias on 2/10/21.
+//
+
+import XCTest
+@testable import Lanchonete
+
+class IngredientCellTests: XCTestCase {
```

Comment:
Maybe add a test that validates that `setQuantity()` works as expected.

Reply:
done